utils/circt-rtl-sim.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
```

Comment:
Please add file header boilerplate.

Also, I think it would make sense to name this `circt-sv-sim.py` since it is likely to be verilog specific, not arbitrary RTL.  If we ever add VHDL, that can be its own tool/script.

Reply:
Actually, mixed language simulations (SystemVerilog and VHDL) are common and very similar. In questa, it's just `vcom` instead of `vlog` so I think it makes sense to support VHDL with the same script.